client/src/components/HistoryViewer/HistoryViewerVersionDetail.js

```diff
@@ -128,6 +126,8 @@ class HistoryViewerVersionDetail extends PureComponent {
       return null;
     }
 
+    const forceDisabled = version.deleted;
```

Comment:
We don't need a whole variable for this (can just use `forceDisabled={version.deleted}` below) - but I won't hold up merging just for that.